vuln/npm/403.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": 403,
+  "title": "Out-of-bounds Read",
+  "overview": "`atob` allocates uninitialized Buffers when number is passed in input on Node.js 4.x and below",
+  "created_at": "2018-03-04",
+  "updated_at": "2018-04-09",
+  "publish_date": "2018-04-09",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "atob",
+  "cves": [],
+  "vulnerable_versions": "<=2.0.3",
+  "patched_versions": ">=2.1.0",
+  "slug": "atob-out-of-bounds-read",
+  "recommendation": "update atob to 2.1.0 or higher",
+  "references": "- https://hackerone.com/reports/321686",
+  "cvss_vector": "CVSS:3.0/AV:P/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:H",
```

Comment:
`AV:P` seems incorrect. `AC:H` is better fit.

Reply:
cc @ChALkeR  ^

Reply:
@lirantal Should be the same as 309, 391, 392, 393.
Those do not require physical access. There is no reason for this to be AC:P.

AC:H because it's hard to actually reproduce this on real apps — attacker-supplied content should end up there, which requires playing around other logic. I.e. it's not just «feed input over http», it's «try to find where user input is fed to `atob` and attempt to supply a typed number there». It requires inpecting other components and, perhaps, bypassing them.